README.md

```diff
@@ -143,6 +143,37 @@ Expose ecto to erlang application
 
 As the ecto interface is based heavily on macros, and not directly invokable in erlang, there should exists reach erlang API to allow to handle and manipulate Ecto model from erlang application. [WiP]
 
+Metrics
+-------
+
+EXD collect some metrics via exometer_core.
+If you want to reporting metrics to reporters you should subscribe each API like this:
```

Comment:
If you want to report metrics to reporters you should subscribe to each API like this: